packages/rest-typings/src/v1/rooms.ts

```diff
@@ -598,6 +598,21 @@ const roomsCleanHistorySchema = {
 
 export const isRoomsCleanHistoryProps = ajv.compile<RoomsCleanHistoryProps>(roomsCleanHistorySchema);
 
+type RoomsOpenProps = {
+	roomId: string;
+};
+
+const roomsOpenSchema = {
+	type: 'object',
+	properties: {
+		roomId: {
+			type: 'string',
```

Comment:
i don't remember but probably something like `nullable: false` or a regex to validate non empty? we must have done similar somewhere else

Reply:
minLength should be enough iirc